pycircleci/api.py

```diff
@@ -747,6 +748,40 @@ def delete_envvar(self, username, project, name, vcs_type=GITHUB):
         resp = self._request(DELETE, endpoint)
         return resp
 
+    def add_context(self, context, username, vcs_type=GITHUB):
+        """Get contexts for an organization
+
+        :param context:  Context name to add.
+        :param username: Org or user name.
+        :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
+
+        Endpoint:
+            POST ``/v2/context``
+        """
+
+        slug = self.owner_slug(username, vcs_type)
+        data = json.dumps({
+            "name": context,
+            "owner": {"id": slug, "type": "organization"}
+        })
+
+        resp = self._request(POST, "context", data=data, api_version=API_VER_V2)
+        return resp
+
+    def get_contexts(self, username, vcs_type=GITHUB):
```

Comment:
how about the other possible args: owner-id, owner-type ?

Reply:
I can look at implementing those. I think maybe they’re not as common, so had been considering keeping it simple for now.

Reply:
Also, do we want to match Circle's term ("list" contexts), or use "get" to match the other get methods in the library?

Reply:
Implemented it - let me know how you think of the style.

Reply:
I'd stick with `get` to be consistent with most of the other similar methods